### PR TITLE
fix(nextjs/ssr): parse `params.asPath`

### PR DESCRIPTION
**Pull request by iam4x**
There is an edge case where `params.asPath === '/'` and gets converted to `{'/': null}` as search state.

Checking that we have an `?` before fix this issue.

### CODE REVIEW

File: packages/react-instantsearch/examples/next-app/pages/index.js [hunk 1]
```diff
@@ -28,9 +28,9 @@ export default class extends React.Component {
      to initialize the searchState. 
   */
   static async getInitialProps(params) {
-    const searchState = qs.parse(
-      params.asPath.substring(params.asPath.indexOf('?') + 1)
-    );
+    const searchState = params.asPath.includes('?')
```
Comment (Haroenv) [inline packages/react-instantsearch/examples/next-app/pages/index.js]: .includes is not supported everywhere we support. I think you should do `indexOf('?') > -1` 😄

Comment (iam4x) [inline packages/react-instantsearch/examples/next-app/pages/index.js]: In this context with nextjs and babel is it safe to use `.includes` 👍

Comment (vvo) [inline packages/react-instantsearch/examples/next-app/pages/index.js]: available in evergreen browsers, always check https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/includes#Browser_compatibility. So it's good

Comment (Haroenv) [inline packages/react-instantsearch/examples/next-app/pages/index.js]: Ah, I forgot that this was node-only

**Pull request merged**